### PR TITLE
Adds an explanation doc for the Monthly Key Metrics report

### PR DESCRIPTION
**Pull request by zachmargolis**
Next steps: a PR in the IDP to link to this document

### CODE REVIEW

File: _articles/monthly-key-metrics-explainer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Monthly Key Metrics Reporting"
+description: "Overview of report, includes metrics and methodology"
+layout: article
+category: "Reporting"
+---
+
+Hello! This document is to help explain the measurements behind the Monthly Key Metrics Report email. If you have any questions, please reach out to Team Agnes in [#login-team-agnes][agnes-slack].
```
Comment (aduth) [inline _articles/monthly-key-metrics-explainer.md:8]: Maybe not worth the effort, but any way we could pull this from `sprint_teams.yml` ?

https://github.com/18F/identity-handbook/blob/b579bd330f066353a1cb4584e163ccb099be15c3/_data/sprint_teams.yml#L16

Comment (aduth) [reply _articles/monthly-key-metrics-explainer.md:8]: Maybe a custom Jekyll filter / tag, for example.

Comment (zachmargolis) [reply _articles/monthly-key-metrics-explainer.md:8]: gave it a shot in c1f7f7a89c891a91e1c0741ccfba49f83171491c

Comment (n1zyy) [reply _articles/monthly-key-metrics-explainer.md:8]: This renders as `[#login-team-agnes][agnes-slack]` in the preview. Is there an additional translation layer not visible here that converts these to proper Markdown links?

File: _articles/monthly-key-metrics-explainer.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+### User counts
+
+Looks at the [total number of users created][query-total-registered]. This is not the [Fully Registered Users][query-fully-registered], which would include users who have created an account as well as set up their MFA.
+
+### Verified Users
+
+Counts currently-active at profile rows. Today, a user can only have one active profile.
+
+{% assign team_agnes = site.data.sprint_teams | find: "name", "Agnes" %}
+[agnes-slack]: {{ team_agnes.slack_channel_url }}
```
Comment (n1zyy) [inline _articles/monthly-key-metrics-explainer.md:26]: I'm game for including this, especially as the pattern is repeated below for other content, but TBH this feels less maintainable to me than just duplicating the link.

File: _articles/monthly-key-metrics-explainer.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/monthly-key-metrics-explainer.md:8]: yeah its a Jekyll filter, this works in the [CG pages preview link](https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.sites.pages.cloud.gov/preview/18f/identity-handbook/margolis-monthly-key-metrics-explainer/articles/monthly-key-metrics-explainer.html)

Comment (n1zyy) [reply _articles/monthly-key-metrics-explainer.md:8]: Nice, TIL! :shipit:

Comment (zachmargolis) [reply _articles/monthly-key-metrics-explainer.md:8]: awesome! can you approve the PR? 🙏

**Review by aduth — APPROVED** [review]
👍

**Pull request merged**